templates/ceph.conf.j2

```diff
@@ -1,5 +1,6 @@
 # {{ ansible_managed }}
 
-[client]
-	client quota = true
-	mon host = {{ os_manila_mount_host }}
+{% for share in os_manila_mount_share_info %}
+[client.{{ share.share_user }}]
```

Comment:
If we have two shares that share the same share user (which is possible within a single project), will this break?